tests/src/test/java/alluxio/client/cli/fs/command/UpdateMountCommandTest.java

```diff
@@ -0,0 +1,184 @@
+          .run("updateMount", "--option", "k1=not_v1", "--option", "k2=v2", mountPoint.toString()));
+      boolean gotException = false;
+      // check mount point is still usable if updateMount fails
+      FileSystemTestUtils.createByteFile(mFileSystem,
+          "/mnt/testFile1", WritePType.CACHE_THROUGH, 20);
+      // check mount point options are not changed
+      Assert.assertEquals(options,
+          mFileSystem.getMountTable().get(mountPoint.getPath()).getProperties());
+    }
+    ImmutableMap<String, String> options2 = ImmutableMap.of("k1", "v3", "k2", "v4");
+    ConfExpectingUnderFileSystemFactory factory2 =
+        new ConfExpectingUnderFileSystemFactory("ufs", options2);
+    try (Closeable closeable =
+             new UnderFileSystemFactoryRegistryRule(factory2).toResource()) {
+      Assert.assertEquals(0, mFsShell
+          .run("updateMount", "--option", "k1=v3", "--option", "k2=v4", mountPoint.toString()));
+      // check mount point is in working state after updated with correct options
+      FileSystemTestUtils.createByteFile(mFileSystem,
+          "/mnt/testFile2", WritePType.CACHE_THROUGH, 20);
+      Assert.assertTrue(mFileSystem.exists(new AlluxioURI("/mnt/testFile2")));
+      URIStatus status = mFileSystem.getStatus(new AlluxioURI("/mnt/testFile2"));
+      Assert.assertTrue(status.isPersisted());
+      // check mount point options are not changed
+      Assert.assertEquals(options2,
+          mFileSystem.getMountTable().get(mountPoint.getPath()).getProperties());
+    }
+  }
+
+  @Test
+  public void updateMountWithSpaceInOptionValues() throws Exception {
+    AlluxioURI mountPoint = new AlluxioURI("/mnt");
+    String ufsPath = mFolder.getRoot().getAbsolutePath();
+    Assert.assertEquals(0, mFsShell
+        .run("mount", "--option", "key=\" value with spaces\"", mountPoint.toString(), ufsPath));
+    Assert.assertEquals(0, mFsShell
+        .run("updateMount", "--option", "key=\" value with spaces 2\"", mountPoint.toString()));
+  }
+
+  @Test
+  public void updateMountWithQuotesInOptionValues() throws Exception {
+    AlluxioURI mountPoint = new AlluxioURI("/mnt");
+    String ufsPath = mFolder.getRoot().getAbsolutePath();
+    Assert.assertEquals(0, mFsShell
+        .run("mount", "--option", "key=valueWith\"Quotes\"", mountPoint.toString(), ufsPath));
+    Assert.assertEquals(0, mFsShell
+        .run("updateMount", "--option", "key=valueWith\"Quotes2\"", mountPoint.toString()));
+  }
+
+  @Test
+  public void updateMountWithEqualsInOptionValues() throws Exception {
+    AlluxioURI mountPoint = new AlluxioURI("/mnt");
+    String ufsPath = mFolder.getRoot().getAbsolutePath();
+    Assert.assertEquals(0,
+        mFsShell.run("mount", "--option", "key=k=v", mountPoint.toString(), ufsPath));
+    Assert.assertEquals(0,
+        mFsShell.run("updateMount", "--option", "key=k=v2", mountPoint.toString()));
+  }
+
+  @Test
+  public void updateRootMountFailed() throws Exception {
```

Comment:
Do you want to validate that it deletes the mount point cleanly?

Reply:
I actually don't know how to get it to fail that bad.
Normally if an option does not work it will be captured in the dry run logic at  https://github.com/Alluxio/alluxio/pull/9107/files#diff-7f82122f923de058e3c4a20b43cd7e4eR2613
The only thing I can think of that will get the mount point deleted is if in the future we accidentally add some check that throws even when `ignoreError` is set to true.